### ISSUE TITLE
UX Ideas

### ISSUE THREAD

**Issue by cstiens** [guardianproject/phoneypot#37]
Hey @n8fr8! You may want to move this to the wiki, but I'm posting it in an issue for now because it's what I have access to. I put together a storyboard with my thoughts about how to tell the story of phoneypot and get people set up when they first use it. This is a start. If you like the direction, I think we should put it in front of some people to get their impressions.

Everything in black is what would be shown on the screen. In green are notes about the interactions. 

![img_0372](https://user-images.githubusercontent.com/1434597/29985914-b652e57c-8f26-11e7-8a08-21536ecb5cc9.JPG)

**Comment by n8fr8** [comment]
Yes, this is excellent. Keep at it!

**Comment by cstiens** [comment]
Here's a link to the next iteration: https://invis.io/Z8DJL004J#/253988213_00

**Comment by cstiens** [comment]
We've done some quick user testing around the office on the last prototype I posted. Here is the feedback so far:

Test 1
* This person said, 'But I always take my phone with me'. We should be clear that one would use a secondary phone for this app.
* In the step where we ask the person to position the phone, we need to show the camera capture view so they can set it up.
* While the person is setting their sound, they need to be able to see the feedback on the screen. Or we could use audio at that step to say 'Talk aloud now'; then 'The test is complete. Come get me.'

Test 2
* We need to make sure the person sets the phone down before setting the sound sensitivity, if we want the interaction the prototype shows
* This person thought that the app was hooked up to a larger security system. We should be clear that the app single-handedly watches the room for you. There's no other hardware required.
* We could also highlight that it's a security system that can travel with you, wherever you go.

Test 3
* After user tuned it to the room, they wanted to leave the phone where they had set it, but still had a couple steps to take.
* It may be better to have the SMS alert step before tuning the room.
* This tester understood what the app what used for.

Test 4
* Tried to slide the audio wave length
* Record activity around a particular device.
* 'Is this your main phone. If so, use a second-hand Android phone to be your eyes and ears instead.'
* Thinks it will record once motion or sound is triggered

**Comment by n8fr8** [comment]
Maybe we need some language like "Turn a spare phone into your personal portable security device" either in the onboarding or generally in the outreach/marketing.

To be clear, the device could also be used, say while you are sleeping in a hotel room, to set off an alarm when the door opens. We need to implement the audible alarm sound as a feature in the case. The point is, there is potential value with even if the person only has one phone.

**Comment by n8fr8** [comment]
To add to that, the other value with one phone, is to defend against "disappearing" of journalists and activists from their homes or rooms. Essentially, again while someone is sleeping, the phone could be setup to monitor the door, and capture anyone coming in, while remotely notifying that persons community or organization. It is harder to communicate all of this in the app itself, so we should likely keep it simple. I just wanted to capture it here as one use case.

**Comment by kngharv** [comment]
The idea of Haven is to use an old, extra Android phone act as a surveillance device.  

The problem with "extra" android phone is that typically, an extra phone doesn't have SIM card inside (hence, extra).  Haven's current design absolutely rely on having a live SIM in order to send out alerts.

Has anyone thought about removing this dependency on live SIM to send out alert (i.e. using wifi).

Thanks in advance

Harv

**Comment by n8fr8** [comment]
@kngharv good point... however, the use of SMS is entirely optional today, even though the UX doesn't really point that out. If you enable the Signal encrypted notifications, that can work entirely over wifi. We are also thinking of enhancing the Tor support to enable syncing between multiple Haven apps.

Overall, we need to improve the onboarding/setup process, to help users without SIM cards or running on tables (kindle fire, etc), to setup the wifi based notification and access features.

**Comment by kngharv** [comment]
@n8fr8 :  My bad.    I registered Signal using Google Voice, but I didn't install Google Voice app at the time, thus, Signal didn't work.  Once I installed Google Voice, Signal worked.   

Now,all I have to do is look for a different phone number service, as the part of world which I travel, Google services is blocked completely, and all public Tor relay are blocked as well.


Other UI experience:

1.  
Setting up remote notification is a bit confusing.   May be it should have two distinct section:

From (haven device Signal number)
current Signal Number:
Register / Verify

To (alert DESIGNATION Signal number)


2. Haven generate a lot of alerts... hundreds if not thousands as a person walk in.   this may be a bit overwhelming.   what is the current ideas on how to make alert more relevant so we don't have to go through everything?   face recognition?   


3. an easy way to delete some of the recorded data?   


4.  keep up with good work.  you guys are awesome. using old Android device as a surveillance device is brilliant!
